.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
 
   artifacts:
     docker:
-      - image: circleci/python:3.7-stretch-node-browsers
+      - image: circleci/python:3.7.6-stretch-node-browsers
```

Comment:
Issues with Node v12.17 when installing optional dependencies - reverting back to previous docker images for all environments.